Formula/tnftpd.rb

```diff
@@ -0,0 +1,45 @@
+class Tnftpd < Formula
+  desc "NetBSD's FTP server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftpd/lukemftpd-51.tar.gz"
+  sha256 "969b8a35fabcc82759da2433973b9606b7b62f73527e76ac8f18d0a19f473c2a"
+
+  keg_only :provided_pre_high_sierra
+
+  depends_on :xcode => :build
+
+  def install
+    system "tar", "zxvf", "tnftpd-20100324.tar.gz"
+
+    cd "tnftpd-20100324" do
+      system "./configure"
+      system "make"
+
+      bin.install "src/tnftpd" => "ftpd"
+      man8.install "src/tnftpd.man" => "ftpd.8"
+      man5.install "src/ftpusers.man" => "ftpusers.5"
+      man5.install "src/ftpd.conf.man" => "ftpd.conf.5"
+      etc.install "examples/ftpd.conf"
+      etc.install "examples/ftpusers"
+    end
+  end
+
+  def caveats
+    <<~EOS.undent
```

Comment:
It's complaining because the syntax is slightly wrong here. You want:
```ruby
def caveats
  <<~EOS
    Blah
    Blah
    Blah
  EOS
end
```
etc.

Reply:
@DomT4 Ok, thanks for clarifying that.